### ISSUE TITLE
Videojs 7.6.0 with angular

### ISSUE THREAD

**Issue by kyawswarthwin** [videojs/video.js#6195]
[06:37:18]  sass: node_modules/video.js/src/css/video-js.scss, line: 5 
            File to import not found or unreadable: ../../node_modules/videojs-font/scss/icons. 

       L5:  @import "../../node_modules/videojs-font/scss/icons";

**Comment by woellij** [comment]
Same here. Not with angular, but with `gulp-sass`

temporary fix: using 7.5.4

**Comment by tsi** [comment]
we are experiencing a similar issue with 7.6.5/6

**Comment by Aniket-Singla** [comment]
I am experiencing the same Issue. Was anyone able to find the solution to the problem??

**Comment by tsi** [comment]
@Aniket-Singla should be fixed in latest 7.7.x
https://github.com/videojs/video.js/pull/6326

I'm not sure what's going on in terms of releases 
official release on GH is still 7.6.x
and npm shows version 7.0.3 as current

**Comment by gkatsev** [comment]
Video.js 7.7.5 is currently latest:
```sh
$ npm dist-tag ls video.js
latest-5: 5.20.5
latest-6: 6.12.1
latest: 7.7.5
next-5: 5.20.5
next-6: 6.13.0
next: 7.7.6
rc: 7.0.0-rc.1
```
